examples/mplot3d/rotate_axes3d_demo.py

```diff
@@ -1,13 +1,21 @@
+'''
+A very simple animation of a rotating 3D plot.
+
+See wire3d_animation_demo for another simple example of animating a 3D plot.
+'''
+
 from mpl_toolkits.mplot3d import axes3d
 import matplotlib.pyplot as plt
-import numpy as np
-
 
 fig = plt.figure()
 ax = fig.add_subplot(111, projection='3d')
+
+# load some test data for demonstration and plot a wireframe
 X, Y, Z = axes3d.get_test_data(0.1)
 ax.plot_wireframe(X, Y, Z, rstride=5, cstride=5)
 
+# rotate the axes and update
 for angle in range(0, 360):
     ax.view_init(30, angle)
     plt.draw()
+    plt.pause(.001)
```

Comment:
If I'm going to make an issue for this, which part is the issue?  That nothing displays for me without the use of plt.pause(), or that the use of plt.pause() triggers that warning?  I'm not really clear on whether plt.draw() is supposed to be sufficient in an example like this.